doc/install.rst

```diff
@@ -4,79 +4,98 @@
 Installing scikit-learn
 =======================
 
-.. note::
+There are different ways to get scikit-learn installed:
 
-    If you wish to contribute to the project, it's recommended you
-    :ref:`install the latest development version<install_bleeding_edge>`.
+  * Install the version of scikit-learn provided by your
+    :ref:`operating system or Python distribution <install_by_distribution>`.
+    This is a quick option for those who have operating systems or Python
+    distributions that distribute scikit-learn.
+    It might not provide the latest release version.
 
+  * :ref:`Install the latest official release <install_official_release>`. This
```

Comment:
Because I used to point the easiest solution first. But I have checked other python packages and they are all in line with what you said. So now (a35cca118d762515f1c27e6b711230125d76fb50) 'the official release' is the first item (and the first paragraph too).